src/Razor/src/Microsoft.CodeAnalysis.Razor.Workspaces/HtmlFactsService.cs

```diff
@@ -3,12 +3,13 @@
 
 using System;
 using System.Collections.Generic;
+using System.Diagnostics.CodeAnalysis;
 using Microsoft.AspNetCore.Razor.Language.Syntax;
 using Microsoft.CodeAnalysis.Text;
 
 namespace Microsoft.VisualStudio.Editor.Razor;
 
-internal abstract class HtmlFactsService
+internal static class HtmlFactsService
```

Comment:
Since this is no longer a service, consider renaming it to `HtmlFacts`.

Reply:
I see you did that :)